taskvine/src/manager/vine_manager.c

```diff
@@ -5772,7 +5779,12 @@ int vine_tune(struct vine_manager *q, const char *name, double value)
 
 	} else if (!strcmp(name, "max-library-retries")) {
 		q->max_library_retries = MIN(1, value);
-
+	} else if (!strcmp(name, "disk-proportion-available-to-task")) {
+		if (value > 1 || value <= 0) {
```

Comment:
If this condition is true, the value should not change. Do not reset to the default.